### ISSUE TITLE
Add File icon to a PF set of icons

### ISSUE THREAD

**Issue by doruskova** [patternfly/patternfly-design#1118]
@mcarrano @mcoker My intern @jhiadlov was working on a Multiple File Upload template and there is a file icon that is not included in our current set of icons. I downloaded it from Font Awesome. However, I guess it should be added to our official set on the website and Design kit.

https://fontawesome.com/v5.15/icons/file?style=solid

[file-solid.svg.zip](https://github.com/patternfly/patternfly-design/files/7802237/file-solid.svg.zip)

**Comment by mcarrano** [comment]
I guess that makes sense. @mceledonia @bdellasc do you see any issue? Here's where this is used: https://www.patternfly.org/v4/components/file-upload---multiple#file-upload-status-expanded

**Comment by bdellasc** [comment]
Looks good to me, @mcarrano. @doruskova. @mcarrano we should just be sure a description is submitted with the icon for inclusion into the icon table. Looks like we've done a pretty good job of that for what's in there so far. Something like: "Represents a file"

Side note @mcarrano: For the status indicator icon in the [icon table](https://www.patternfly.org/v4/guidelines/icons), I noticed we have a "Tooltip" in the description. Is there an effort to add tooltip info for every icon in the icon table?

**Comment by mcarrano** [comment]
@bdellasc No, we don't currently have any effort or plan to do that. Do you think it is necessary or realistic to come up with a recommended tooltip for every icon?

**Comment by bdellasc** [comment]
@mcarrano I guess that effort would fall under "accessibility" and/or "content strategy"? I don't know where the line is between the toolkit providing this stuff vs the consumers of the toolkit adding tooltips into their implementation that uses the toolkit? Not sure if there have been any decisions around those sorts of topics, because the line is blurry there, for sure.

**Comment by mcarrano** [comment]
In any case, the toolkit does not add these tooltips automatically. They are only presented on this page as a guideline or recommendation. Part of the issue is that for many of these icons, the appropriate tooltip is dependent on context making it difficult to come up with a "one size fits all" recommendation.

**Comment by bdellasc** [comment]
@mcarrano that makes sense. Thanks for following up!

**Comment by mcarrano** [comment]
Going to transfer this to patternfly-org to update the website. @doruskova you may need a separate issue to add this to the designkit. This should be a standard font awesome icon.
